templates/zerver/help/import-data-from-slack.md

```diff
@@ -25,3 +25,21 @@ If the users are not from the same organization, you should change your organiza
 {!admin.md!}
 
 2. Disable the **Restrict new users to the following email domains** option.
+
+## Elements that are not mapped one to one
+
```

Comment:
I would reorder these as:
* Avatar
* Attachments (this includes pinned to a channel .. so don't need a separate line for it)
* Reactions 
* Message edit history
* Permission hierarchy